README.md

```diff
@@ -1,79 +1,105 @@
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
-    
+# Deploy On-Premises Web App
+
+## Index <!-- omit in toc -->
+
+- [Inputs](#inputs)
+- [Prerequisites](#prerequisites)
+- [Example](#example)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
 
 ## Inputs
-| Parameter | Is Required | Description           |
-| --------- | ----------- | --------------------- |
-| `input-1` | true        | Description goes here |
-| `input-2` | false       | Description goes here |
 
-## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
+| Parameter                  | Is Required | Description                                                                       |
+| -------------------------- | ----------- | --------------------------------------------------------------------------------- |
+| `server`                   | true        | The name of the target server                                                     |
+| `service-account-id`       | true        | The service account name                                                          |
+| `service-account-password` | true        | The service account password                                                      |
+| `source-zip-file-path`     | true        | The path to the zip file that contains the application files                      |
+| `deployment-folder-path`   | true        | The path to deploy the application files to                                       |
+| `clean-deployment-folder`  | false       | Remove files that are not in the source zip file, accepts values of true or false |
+| `server-public-key`        | true        | Path to remote server public ssl key (local path like c:\folder)                  |
+
+## Prerequisites
+
+The On-Premises Web Application Deployment action uses Web Services for Management, [WSMan], and Windows Remote Management, [WinRM], to create remote administrative sessions. Because of this Self-Hosted, Windows OS GitHubs Actions Runners, `runs-on: [self-hosted, windows-2019]`, must be used.  This will also allow the actions runner access to our on-premises applications servers.
```

Comment:
If someone outside of our org stumbles across this they're going to read that self-hosted is required.  Would that be true for them?  Is this only going to work on our own customized action runners?

If so I might put another disclaimer saying this action has been customized for im-open's use and may not work outside of our organization.

Reply:
I'll try to clarify in a generic way.